src/terra-dev-site/guide/WhatIsTerra.a.guide.mdx

```diff
@@ -5,11 +5,11 @@ import MedicineIcon from '../icon/MedicineIcon';
 <div className="tui-illustration-grid">
 <div className="tui-illustration-grid-col">
 
-Build and integrate apps with Cerner.
+Terra is Cerner's open-source platform for creating web applications integrating into the Cerner application ecosystem. Built on a foundation of [React](https://reactjs.org/), Terra provides the building blocks to construct accessible single-page applications. More than just a component library, Terra offers solutions to common application problems such as internationalization, modal presentation, unsaved changes, loading overlays, and themes.
 
-Terra is Cerner's open-source platform for creating web applications integrating into the Cerner application ecosystem. Built on a foundation of [React](https://reactjs.org/), Terra provides the building blocks to construct accessible single-page applications. More than just a component library, Terra offers solutions to common application problems such as internationalization, modal presentation, unsaved changes, loading overlays, and themes. Terra also offers a reuseable set of configurable react components to help build out applications. Similar to Apple's [appkit](https://developer.apple.com/documentation/appkit), these [React components](https://engineering.cerner.com/terra-ui/components) are designed to be building blocks that are composed into higher-order workflows and components to represent your concepts.
+Terra also offers a reuseable set of configurable React components to help build out applications. Similar to Apple's [appkit](https://developer.apple.com/documentation/appkit), these [React components](https://engineering.cerner.com/terra-ui/components) are designed to be building blocks that are composed into higher-order workflows and components to represent your concepts. Terra components are built to be accessible and meet Cerners exacting UX design standards.
```

Comment:
should exacting be existing?

Reply:
nope, i mean exacting :)

Reply:
I can't speak for @mjhenkes but I think he meant exacting:
![image](https://user-images.githubusercontent.com/4873279/77762773-05f1bf80-7008-11ea-9cd6-144197c3438f.png)

Reply:
![](https://media.giphy.com/media/B0vFTrb0ZGDf2/giphy.gif)